ui/src/views/Settings.vue

```diff
@@ -22,16 +22,283 @@
+                        <cv-tooltip
+                          alignment="start"
+                          direction="bottom"
+                          :tip="$t('settings.admin_login_tips')"
+                          class="info mg-bottom"
+                        >
+                        </cv-tooltip>
+                      </span>
+                      <span>:</span>
+                      <cv-link
+                        class="mg-bottom mg-left"
+                        :href="'https://' + hostname + ':5280/admin/'"
+                        target="_blank"
+                        :inline="false"
+                      >
+                        {{ $t("settings.webadmin_link") }}
+                      </cv-link>
+                    </template>
+                    <cv-text-area
+                      :label="$t('settings.adminList')"
+                      v-model.trim="adminsList"
+                      :invalid-message="error.adminsList"
+                      :helper-text="$t('settings.Write_administrator_list')"
+                      :value="adminsList"
+                      class="maxwidth textarea"
+                      ref="adminsList"
+                      :placeholder="$t('virtualhosts.write_one_user_per_line')"
+                      :disabled="
+                        loading.getConfiguration || loading.configureModule
+                      "
+                    >
+                    </cv-text-area>
+                  </template>
+                  <cv-toggle
+                    value="s2s"
+                    :label="$t('settings.Enable_federation_s2s')"
+                    v-model="isS2sEnabled"
+                    :disabled="
+                      loading.getConfiguration || loading.configureModule
+                    "
+                    class="mg-bottom"
+                  >
+                    <template slot="text-left">{{
+                      $t("settings.disabled")
+                    }}</template>
+                    <template slot="text-right">{{
+                      $t("settings.enabled")
+                    }}</template>
+                  </cv-toggle>
+                  <cv-toggle
+                    value="mod_mam_status"
+                    :label="
+                      $t('settings.Enable_message_archive_management_mod_mam')
+                    "
+                    v-model="isModMamStatus"
+                    :disabled="
+                      loading.getConfiguration || loading.configureModule
+                    "
+                    class="mg-bottom"
+                  >
+                    <template slot="text-left">{{
+                      $t("settings.disabled")
+                    }}</template>
+                    <template slot="text-right">{{
+                      $t("settings.enabled")
+                    }}</template>
+                  </cv-toggle>
+                  <template v-if="isModMamStatus">
+                    <cv-toggle
```

Comment:
I would invert this logic: instead of using `CvToggle` I would use a `NsSlider` with label `Keep old messages for (days)`. It has support for forever/unlimited. See `https://github.com/NethServer/ns8-core/blob/main/core/ui/src/stories/NsSlider.stories.js` or try it live on storybook

Reply:
done